### PR TITLE
SVC voltage remote control

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
#42 

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
Static var compensator voltage remote control is not supported.

**What is the new behavior (if this is a feature change)?**
Static var compensator voltage remote control is now supported, in a same way than generators.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java [hunk 1]
```diff
@@ -70,26 +70,30 @@ public void visitThreeWindingsTransformer(ThreeWindingsTransformer transformer,
...
-                            double previousTargetV = generator.getTargetV();
-                            double remoteNominalV = generator.getRegulatingTerminal().getVoltageLevel().getNominalV();
-                            double localNominalV = generator.getTerminal().getVoltageLevel().getNominalV();
+                            double remoteNominalV = regulatingTerminal.getVoltageLevel().getNominalV();
+                            double localNominalV = injection.getTerminal().getVoltageLevel().getNominalV();
                             scaleV = localNominalV / remoteNominalV;
-                            LOGGER.warn("Generator remote voltage control is not yet supported. The voltage target of " +
-                                            "generator {} with remote control is rescaled from {} to {}",
-                                    generator.getId(), previousTargetV, previousTargetV * scaleV);
+                            LOGGER.warn("Remote voltage control is not yet supported. The voltage target of " +
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java]: Maybe change the comment just to say that remote voltage control is not active, so targets will be rescaled.

**Comment by geofjamg** [comment]
> I understand that we will use the parameter `voltageRemoteControl` to activate voltage remote control for generators and SVCs, in the same time. I don't know if it could be useful to have 2 parameters instead of just one.

Ok we can keep the 2 parameters.

**Comment by geofjamg** [comment]
> An other question, about the name chosen : `voltageRemoteControl` is not so precise and we will have to change it when we will model voltage remote regulation for shunt and tap changers. So, we keep like that and we change later or it is better to decide now ?

As this  PR will only be merge when core 3.2.0 will be released, we can prepare in another PR all needed options and throws an exception if not yet supported. It would be better to add all this parameters to LoadFlowParameters and not to OLF specific parameter. That would be a good idea to do it for core 3.2.0.

File: src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java (hunk 1 above)
Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java]: Yes this warning has  been added while not yet implemented but now it has to be fixed "yet supported" => "active"

**Pull request merged**